workflow-templates/im-deploy-az-app-manually.yml

```diff
@@ -103,34 +110,78 @@ jobs:
           scope: ${{ env.ENVIRONMENT }}
           create-output-variables: true
         env:
-          # This variable is used to swap slots and annotate app insights
-          RESOURCE_GROUP@dev: ''
-          RESOURCE_GROUP@qa: ''
-          RESOURCE_GROUP@stage: ''
-          RESOURCE_GROUP@demo: ''
-          RESOURCE_GROUP@uat: ''
-          RESOURCE_GROUP@prod: ''
+          # Resource group you are targeting for deploy.  Also this variable is used to delete and re-create azure locks.
```

Comment:
What if these comments above the target and primary indicated they should include primary and secondary regions for target which are generally na26 and na27, and the second comment indicated it should generally be the na26 region.  I know if you sit and look at the values you can fill out you would figure that out, but I did that yesterday and still got them backwards.

Reply:
It looks like these variables didn't get renamed to `TARGET_RESOURCE_GROUP` at the same time the corresponding ones in the `...tf-manual-apply` workflow did.

Reply:
I will update the comment to suggest the na26/na27 regions